jobs/config.json

```diff
@@ -415,6 +415,141 @@
       "UNKNOWN"
     ]
   }, 
+  "ci-kubernetes-e2e-gce-cos-beta-k8s-current": {
+    "args": [
+      "--env-file=jobs/platform/gce.env", 
+      "--env-file=jobs/k8s/k8s_current.env", 
+      "--env-file=jobs/image/cos_beta.env", 
+      "--env-file=jobs/suite/default.env", 
+      "--env-file=jobs/ci-kubernetes-e2e-gce-cos-beta-k8s-current.env", 
+      "--mode=local", 
+      "--timeout=150m"
+    ], 
+    "scenario": "kubernetes_e2e", 
+    "sigOwners": [
+      "sig-node"
+    ]
+  }, 
+  "ci-kubernetes-e2e-gce-cos-beta-k8s-current-serial": {
```

Comment:
IIRC, the project name cannot exceed 30 characters. Since composing the project names in this way is not scalable, should we use some hash instead?